docs/testing.md

```diff
@@ -179,6 +179,7 @@ module.exports = {
     '^.+\\.(jpg|jpeg|png|gif|webp|svg)$': '<rootDir>/__mocks__/fileMock.js',
   },
   testPathIgnorePatterns: ['<rootDir>/node_modules/', '<rootDir>/.next/'],
+  testEnvironment: 'jsdom',
```

Comment:
Not a major issue. What do you think about setting the environment locally (per file) instead of globally as [recommended by Jest](https://jestjs.io/blog/2021/05/25/jest-27)?

Reply:
I've gone back and forth on that recommended pattern myself in real life projects 😂 

I think either way could work, but I'd be happy to modify this to follow the JSDoc approach if it is this project's preference?

Reply:
Ok 👍  Let's go with per file since not every test will require `jsdom` and follows Jest recommendation. I added a note in [docs here](https://github.com/vercel/next.js/pull/28209/files#diff-c22885d97fa21ad974a3f5f982d92b6f45edc251fd34bc14223492ebd7391245R306s) so people can set it globally if they want to.

Reply:
@delbaoliveira I made the changes. I think there may be merge conflicts with the changes in #28209 though.

Reply:
Thank you and yes, there will be some merge conflicts. That's ok though, I can update the other pull request! 🙏🏼